### PR TITLE
Tests

### PR DESCRIPTION
**Pull request by lubegasimon**

### CODE REVIEW

File: contact.ts [hunk 1]
```diff
@@ -5,7 +5,7 @@ import * as prompt from 'inquirer';
 
 import * as addContact from './controllers';
 
-const questions = [
+const questions: any[] = [
```
Comment (epicallan) [inline contact.ts]: any is not the proper type for questions array. Write a non any type

File: __tests__/controllers.test.ts [hunk 2]
```diff
@@ -0,0 +1,95 @@
+import { lowerCase } from '../controllers';
+import { upperCase } from '../controllers';
+import { addContact } from '../controllers';
+
+
+test('test if lowerCase function runs', () => {
```
Comment (epicallan) [inline __tests__/controllers.test.ts]: test message should probably say. `lowerCase functions throws error if not passed an argument.` but again I find this to be an  unnecessary test

File: __tests__/controllers.test.ts [hunk 3]
```diff
@@ -0,0 +1,95 @@
...
+      } else {
+        return {
+          message: () => `expected ${received} to be divisible by ${argument}`,
+          pass: false,
+        };
+      }
+    },
+  });
+  
+  test('even and odd numbers', () => {
```
Comment (epicallan) [inline __tests__/controllers.test.ts]: Not necessary tests.

File: __tests__/controllers.test.ts [hunk 4]
```diff
@@ -0,0 +1,95 @@
...
+
+expect.extend({
+    toReturnLowerCasedValue(email: string): string{
+        if(email.toLowerCase()){
+
+        }
+    }
+})
+
+expect.extend({
```
Comment (epicallan) [inline __tests__/controllers.test.ts]: Good to know how to extend expect but not necessary for this current test suite.

File: package.json [hunk 5]
```diff
@@ -4,7 +4,8 @@
   "description": "Contact Management System (phone book)",
   "main": "index.js",
   "scripts": {
-    "test": "echo \"Error: no test specified\" && exit 1"
+    "start": "ts-node contact.ts",
```
Comment (epicallan) [inline package.json]: 👍

File: __tests__/controllers.test.ts (hunk 4 above)
Comment (lubegasimon) [reply __tests__/controllers.test.ts]: yea, i read about it

File: __tests__/controllers.test.ts (hunk 2 above)
Comment (lubegasimon) [reply __tests__/controllers.test.ts]: noted!

File: contact.ts (hunk 1 above)
Comment (lubegasimon) [reply contact.ts]: resolved

**Comment by epicallan** [comment]
@lubegasimon Merge Pull Request if you feel you have addressed everything.

**Comment by lubegasimon** [comment]
Putting the bugs aside, i think everything is fine, or @fairly fine

**Pull request merged**